pkg/devspace/services/synccontroller/controller.go

```diff
@@ -226,10 +233,12 @@ func (c *controller) startSync(options *Options, onInitUploadDone chan struct{},
 	)
 
 	localPath := "."
+	if runtime.GOOS == "darwin" {
+		localPath, _ = realWorkDir()
```

Comment:
Either return the error, or make sure the local path is "." again. Please also move this logic into the function

Reply:
done